docs/concepts/staking/block_reward.md

```diff
@@ -1,21 +1,70 @@
 ---
 tags:
+  - block reward
   - Staking
 ---
 
 
 # Staking
 
 
-## Block reward
+## Block Reward
+---
+
+Block rewards in a blockchain serve as an incentive for validators who actively contribute to the creation of new blocks within the blockchain network. This rewarding process, commonly referred to as validation, is particularly relevant in **delegated proof-of-stake (DPoS)** blockchains.
```

Comment:
The actual distribution of rewards happens as a step separate from the act of validation, so the phrase "rewarding process, commonly referred to as validation", doesn't seem quite accurate? Maybe omit this qualifier.